settings/common.py

```diff
@@ -348,7 +350,7 @@
     "ALGORITHM": "HS256",
     "AUDIENCE": None,
     "ISSUER": None,
-    "AUTH_HEADER_TYPES": ("Bearer",),
```

Comment:
Why are we changing this?

Reply:
This is changed so that `rest_auth` can verify the user after login. If we don't make this change, we'll need to make changes in the frontend to set default `Authorization` headers as `Bearer <Token>` (currently, it sets default `Authorization` header as `Token <Token>`, [here](https://github.com/Cloud-CV/EvalAI/blob/adde7be63e25af834d65ef2625103040122f561e/frontend_v2/src/app/services/api.service.ts#L26))